airflow/www/app.py

```diff
@@ -338,7 +338,10 @@ def chart_data(self):
         pd.set_option('display.max_colwidth', 100)
         hook = db.get_hook()
         try:
-            df = hook.get_pandas_df(wwwutils.limit_sql(sql, CHART_LIMIT))
+            if db.conn_type == 'mssql':
```

Comment:
I'd prefer passing the conn_type to `limit_sql` and have the logic in there. 

A slightly more ambitious refactor would be to move `limit_sql` to the hook class, it would probably be set in DbApiHook as a LIMIT clause and overridden where needed (mssql, eventually OracleHook, ... ).